.idea/compiler.xml

```diff
@@ -20,6 +20,7 @@
   <component name="JavacSettings">
     <option name="ADDITIONAL_OPTIONS_OVERRIDE">
       <module name="Silverpricing-backend" options="-parameters" />
+      <module name="silverpricing-ehpad-backend" options="-parameters" />
```

Comment:
:information_source: Module configuration present in an unexpected file (.idea/compiler.xml) (Severity: Minor)